src/styles/styles.css

```diff
@@ -0,0 +1,178 @@
+html {
+  font-family: Roboto, Arial, 'Helvetica Neue', 'Segoe UI', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+}
+
+.card__content {
+  width: 198px;
+  height: 406px;
+  border-radius: 5px;
+}
```

Comment:
The `.card__content` class has the same fixed width and height as `.card`. If `.card__content` is meant to be the same size as `.card`, it's better to use inheritance or relative units instead of redeclaring the same dimensions.